programmers/김서방-찾기.js

```diff
@@ -0,0 +1,4 @@
+function solution(seoul) {
+    var i = seoul.indexOf("Kim");
+    return `�輭��� ${i}�� �ִ�`;
+}
```

Comment:
```suggestion
function solution(seoul) {
    return `김서방은 ${seoul.indexOf("Kim")}에 있다`
}
```
굳이 i를 변수로 선언하지 않고 위 코드처럼 한방에 가능해질 수 도 있습니다.